spec/rack_spec.rb

```diff
@@ -31,4 +31,11 @@ def app
     expect(last_request.is_crawler?).to eq(true)
     expect(last_request.crawler_name).to eq("Test Bot")
   end
+
+  it "should be thread safe" do
+     instance = described_class.new(app)
+     instance.call({})
+     expect(instance.instance_variable_get(:@env)).to be_nil
```

Comment:
useless spec 🤔

Reply:
Right, I should've removed that, I added it in the previous commit when I just used `dup`